packages/mui-material/test/typescript/hoc-interop.spec.tsx

```diff
@@ -21,22 +21,22 @@ const filledProps = {
 
 // baseline behavior
 <TextField variant="filled" {...filledProps} />;
-// @ts-expect-error
+
 <TextField {...filledProps} />; // desired to throw
```

Comment:
Can we remove the comments also? Same in other places.
```suggestion
<TextField {...filledProps} />;
```

Reply:
fixed in this commit https://github.com/mui/material-ui/pull/36737/commits/74d32dac6989da9164fb6bffddc67e9b735a148a